Task3/Task3/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task3
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите время, на которое указывает часовая стрелка (1-12):");
+            double hour = double.Parse(Console.ReadLine()); 
+            Console.WriteLine("Введите время, на которое указывает минутная стрелка (0-59):");
+            double minute = double.Parse(Console.ReadLine());
```

Comment:
В задании указано, что для данных (часы и минуты) нужно использовать целочисленный тип данных.
```suggestion
            Console.WriteLine("Введите время, на которое указывает часовая стрелка (1-12):");
            int hour = int.Parse(Console.ReadLine()); 
            Console.WriteLine("Введите время, на которое указывает минутная стрелка (0-59):");
            int minute = int.Parse(Console.ReadLine());
```